aiogmaps/client.py

```diff
@@ -102,22 +112,29 @@ def _get_params(self, params, accepts_clientid=False):
             base_url = self.base_url
 
         base_url = URL(base_url)
-
-        params = self._get_params(params)
+        params = self._get_params(
+            path='/' + url.lstrip('/'),
+            params=params,
+            accepts_clientid=accepts_clientid
+        )
 
         if aiohttp3:
             # https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientSession.request
             kwargs.update({'ssl': self.verify_ssl})
 
+        combined_url = URL(
+            "?".join([str(base_url / url.lstrip('/')),
+                      urlencode_params(params)]),
```

Comment:
I tried with - still 403 because yarl encodes params ;/
```
combined_url = URL(str(base_url.with_path(url).with_query(urlencode_params(params))), encoded=True)

```

Reply:
It is because you encode signature in `_get_params` as well, you should encode params but not signature like here https://github.com/googlemaps/google-maps-services-python/blob/master/googlemaps/client.py#L321-L322

Reply:
From my observation, the problem is with lists. For example:
location=(39.603481,-119.682251) is transformed by urlencode_params to `location=39.603481%2C-119.682251` and this value is used for signature calculation.
So the proper request should be:
`https://maps.googleapis.com/maps/api/timezone/json?location=39.603481%2C-119.682251 ....`
But if we pass params to yarl.URL or we will not use encoded=True flag we receive:
`https://maps.googleapis.com/maps/api/timezone/json?location=39.603481,-119.682251 ....`
This causes that signature verification fails.

Reply:
It looks like urlencode_params function has no impact on signature encoding

Reply:
Hm, seems like yes and looks like it is expected by `yarl` to not encode `,` (btw chrome do not do so as well). Is such case i recommend to refactor `get_params` to `get_authed_url` similar to [here](https://github.com/googlemaps/google-maps-services-python/blob/master/googlemaps/client.py#L294) to not encode params twice.